publish.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash -ex
 
 git pull origin master --tags
-npx lerna exec npm install
-npx lerna bootstrap
-npx lerna run build
 
 npx lerna version --no-push
 git push origin --tags
```

Comment:
maybe best to just delete this file now?

Reply:
Yup agree, maybe we can merge this now?